### PR TITLE
DDCE-1017:Content Updates for Section 6 - Bank Details

### PR DESCRIPTION
**Pull request by nathan-rees**
# DDCE-(StoryNumberHere)

##Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/views/operationsAndFunds/CharitableObjectivesView.scala.html [hunk 1]
```diff
@@ -48,7 +48,6 @@
                 maxLength = Some(255),
                 isPageHeading = false,
                 label = messages("charitableObjectives.label"),
-                hint = Some(Html(messages("charitableObjectives.hint"))),
```
Comment (abhijeetardale) [inline app/views/operationsAndFunds/CharitableObjectivesView.scala.html:51]: changes should be for bank details pages

File: conf/messages.cy [hunk 2]
```diff
@@ -563,21 +563,20 @@ startCharitableObjectives.p2 = (W)Adding information about objectives and purpos
 
 # CharitableObjectivesPage Messages
 # ----------------------------------------------------------
-charitableObjectives.title = (W)What are your charity''s charitable objectives?(W)
-charitableObjectives.heading = (W)What are your charity''s charitable objectives?(W)
-charitableObjectives.error.required = (W)Enter your charity''s charitable objectives(W)
+charitableObjectives.title = (W)What are the charity''s charitable objectives?(W)
```
Comment (abhijeetardale) [inline conf/messages.cy:566]: changes should be for bank details pages

File: app/views/operationsAndFunds/PublicBenefitsView.scala.html [hunk 3]
```diff
@@ -24,8 +24,7 @@
     formHelper: FormWithCSRF,
     errorSummary: components.errorSummary,
     button: components.button,
-    inputCharacterCount: components.inputCharacterCount,
-    fieldset: components.fieldset
+    inputCharacterCount: components.inputCharacterCount
```
Comment (abhijeetardale) [inline app/views/operationsAndFunds/PublicBenefitsView.scala.html:27]: changes should be for bank details pages

File: conf/messages.en [hunk 4]
```diff
@@ -563,21 +563,20 @@ startCharitableObjectives.p2 = Adding information about objectives and purposes
 
 # CharitableObjectivesPage Messages
 # ----------------------------------------------------------
-charitableObjectives.title = What are your charity''s charitable objectives?
-charitableObjectives.heading = What are your charity''s charitable objectives?
-charitableObjectives.error.required = Enter your charity''s charitable objectives
+charitableObjectives.title = What are the charity''s charitable objectives?
```
Comment (abhijeetardale) [inline conf/messages.en:566]: changes should be for bank details pages

**Review by abhijeetardale — CHANGES_REQUESTED** [review]

**Comment by robert-webb-hmrc** [comment]
This looks like there's a conflict in git history here, these changes have already been merged in.

**Pull request closed without merging**